gramming/templates/index.html

```diff
@@ -0,0 +1,11 @@
+<!-- wp:template-part {"slug":"header","area":"header"} /-->
+
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:query {"queryId":8,"query":{"perPage":"12","pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false,"taxQuery":{"category":[5]}},"displayLayout":{"type":"flex","columns":3},"align":"full","className":".add-remove-space"} -->
```

Comment:
Can we remove this class: `.add-remove-space`? I can't find the corresponding CSS.

Reply:
Indeed.  Removed.

I checked the source theme (block canvas) and didn't see it there so I'm not sure where it came from.